modules/apps/commerce/commerce-product-item-selector-web/src/main/resources/META-INF/resources/definition_item_selector.jsp

```diff
@@ -92,10 +94,10 @@ String itemSelectedEventName = cpDefinitionItemSelectorViewDisplayContext.getIte
 				value="<%= HtmlUtil.escape(cpDefinitionItemSelectorViewDisplayContext.getSku(cpDefinition, locale)) %>"
 			/>
 
-			<liferay-ui:search-container-column-date
+			<liferay-ui:search-container-column-text
 				cssClass="table-cell-expand"
 				name="modified-date"
-				property="modifiedDate"
+				value="<%= HtmlUtil.escapeJS(dateFormatDateTime.format(cpDefinition.getModifiedDate())) %>"
```

Comment:
Can you please wrap this into a new method of the dispaly context?

Thank you!

Reply:
It should be good now. See new method @andrea-ale-sbarra